tests/requests/tests.py

```diff
@@ -246,11 +246,14 @@ def test_far_expiration(self):
 
     def test_max_age_expiration(self):
         "Cookie will expire if max_age is provided"
-        response = HttpResponse()
-        response.set_cookie('max_age', max_age=10)
+        current_time = time.time()
+        with freeze_time(current_time):
+            response = HttpResponse()
+            response.set_cookie('max_age', max_age=10)
```

Comment:
Technically, yes, but it looks a bit weird:

```py
def test_foo(self):
    current_time = time.time()
    response = HttpResponse()
    with freeze_time(current_time):
        response.set_cookie('max_age', max_age=10)
        # ...
    # ...
```

See the other usages of the helper. The whole test block is within the context manager in the following tests:

* https://github.com/django/django/blob/fcd08c175787e909b3eb98f756317a07741c48dd/tests/signing/tests.py#L134
* https://github.com/django/django/blob/fcd08c175787e909b3eb98f756317a07741c48dd/tests/signed_cookies_tests/tests.py#L48